app/src/main/java/com/globant/calculatoronboardingkotlin/mvp/view/CalculatorView.kt

```diff
@@ -1,4 +1,21 @@
 package com.globant.calculatoronboardingkotlin.mvp.view
 
-class CalculatorView {
+import android.app.Activity
+import kotlinx.android.synthetic.main.activity_main.calculation_label
+import kotlinx.android.synthetic.main.activity_main.input_label
+
+class CalculatorView(activity: Activity) : ActivityView(activity) {
+
+    fun showResult(result: String?) {
+        activity?.calculation_label?.text = result
```

Comment:
you should declare the calculation label as a private val and here just use that variable

Reply:
done, thanks